libcxx/src/string.cpp

```diff
@@ -247,6 +248,36 @@ long double stold(const wstring& str, size_t* idx) { return as_float<long double
 
 // to_string
 
+#if _LIBCPP_STD_VER >= 26
```

Comment:
It seems necessary to change the `<string>` header to support versioning. Perhaps we also need make new version of these functions provided in an inline namespace as the old ones are `_LIBCPP_EXPORTED_FROM_ABI`.